src/types/Client.d.ts

```diff
@@ -6,7 +6,7 @@ import { ExprArg } from './query'
 import RequestResult from './RequestResult'
 import { Subscription, SubscriptionEventHandlers } from './Stream'
 
-type StreamEventFields = 'action' | 'document' | 'diff' | 'prev'
+type StreamEventFields = 'action' | 'document' | 'diff' | 'prev' | 'index'
```

Comment:
Any way to make the stream event fields available to non-TS devs?

Reply:
They are available as string arguments.

```js
client.stream(.., { fields: ['document', 'index'] })
```